Makefile

```diff
@@ -43,4 +43,47 @@ thrift-image:
 protocompile:
 	$(PROTOTOOL) prototool compile proto --dry-run
 
-.PHONY: test-ci clean thrift thrift-image $(THRIFT_FILES) swagger-validate protocompile
+JAEGER_DOCKER_PROTOBUF ?= jaegertracing/protobuf:0.1.0
+PROTOC := docker run --rm -u ${shell id -u} -v${PWD}:${PWD} -w${PWD} ${JAEGER_DOCKER_PROTOBUF} --proto_path=${PWD}
+PROTO_INCLUDES := \
+	-Iproto/api_v2 \
+	-I/usr/include/github.com/gogo/protobuf
+# Remapping of std types to gogo types (must not contain spaces)
+PROTO_GOGO_MAPPINGS := $(shell echo \
+		Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/types, \
+		Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types, \
+		Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types, \
+		Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types, \
+		Mgoogle/api/annotations.proto=github.com/gogo/googleapis/google/api, \
+		Mmodel.proto=github.com/jaegertracing/jaeger/model \
+	| sed 's/ //g')
+
+.PHONY: proto
+proto:
+	mkdir -p proto-gen
+	$(PROTOC) \
+		$(PROTO_INCLUDES) \
+		--gogo_out=plugins=grpc,$(PROTO_GOGO_MAPPINGS):$(PWD)/proto-gen \
```

Comment:
if think we should be trying to build for multiple languages (see #41)

Reply:
+1 in fact I didn't know it can be generated into multiple lags using a single command

Reply:
I have made the directories to varios langs configurable. The makefile could be loaded in jaeger and just changed the output dirs